### PR TITLE
Fix flow and export typings

### PR DESCRIPTION
**Pull request by davidyaha**

### CODE REVIEW

File: packages/server/src/emailTemplates.js [hunk 1]
```diff
@@ -1,4 +1,16 @@
 // @flow
+import type { UserObjectType } from '@accounts/common';
+
+export type EmailTemplateType = {
+  subject: (user?: UserObjectType) => string,
```
Comment (pradel) [inline packages/server/src/emailTemplates.js]: You can add an optional `from` parameter `from?: string`

Comment (davidyaha) [inline packages/server/src/emailTemplates.js]: Will it be used by the package?

Comment (pradel) [inline packages/server/src/emailTemplates.js]: It is already implemented

**Comment by TimMikeladze** [comment]
👍

**Pull request merged**